src/components/Sidebar/Sidebar.tsx

```diff
@@ -109,7 +153,39 @@ const Sidebar: FC = () => {
                 value=""
               />
             )}
-
+            {nodeType === 'reference' && (
+              <div>
+                <DropDown
+                  id="nodeType"
+                  label="Node Type"
+                  options={nodeTypeOptions}
+                  onChange={selectNodeType}
+                  value="Reference"
+                />
+                <div className={styles.referenceDropdown}>
+                  <DropDown
+                    id="reference"
+                    label="Pathway Reference"
+                    options={pathwayOptions}
+                    onChange={selectPathwayReference}
+                    value={(currentNode as ReferenceNode).referenceId}
+                  />
+                  <div className={styles.referenceChevron}>
+                    <Link
```

Comment:
I noticed the node redirects in the sidebar just use `history.push(url)` instead of this `Link` component. Unless there's a reason to use this over the redirect I think we should stay consistent. See `utils/nodeUtils.tsx redirect()` for how redirecting nodes works